docs/source/api_commands.rst

```diff
@@ -276,901 +774,1168 @@ Return example::
+get_server_timer
+~~~~~~~~~~~~~~~~
 
 Description:
 
 Parameters:
 
 
-start_wash_then_charge
-----------------------
+set_server_timer
+~~~~~~~~~~~~~~~~
 
 Description:
 
 Parameters:
 
+get_timezone
+~~~~~~~~~~~~~~~~~
 
-status
-------
-
-Get: get_status
-
-Description: Get status information of the device.
+Description: Get the device's time zone.
 
 Parameters: None
 
-Returns:
-
-    msg_ver:
-
-    msg_seq:
+Returns: Time zone by the TZ identifier (e.g., America/Los_Angeles)
 
-    state:
+======================  =========
+Vacuum Model            Supported
+======================  =========
+Roborock S7 MaxV Ultra  Yes
+Roborock S8 Pro Ultra   Yes
+======================  =========
 
-    battery: Battery level of your device.
 
-    clean_time: Total clean time in hours.
+set_timezone
+~~~~~~~~~~~~~~~~~
 
-    clean_area: Total clean area in meters.
+Description: Sets the device's time zone
 
-    error_code:
+Parameters: 
 
-    map_reset:
 
-    in_cleaning:
 
-    in_returning:
 
-    in_fresh_state:
 
-    lab_status:
+Sound
+------------
 
-    water_box_status:
+get_sound_volume
+~~~~~~~~~~~~~~~~
 
-    back_type:
+Description: Returns the volume of the sound played by the vacuum
 
-    wash_phase:
+Parameters: None
 
-    wash_ready:
+Returns: 
 
-    fan_power:
+    volume: The volume of the sound played by the vacuum
 
-    dnd_enabled:
+Example::
 
-    map_status:
+    72
 
-    is_locating:
+======================  =========
+Vacuum Model            Supported
+======================  =========
+Roborock S8 Pro Ultra   Yes
+======================  =========
 
-    lock_status:
+change_sound_volume
+~~~~~~~~~~~~~~~~~~~
 
-    water_box_mode:
+Description: Sets the volume of the sound played by the vacuum
 
-    water_box_carriage_status:
+Parameters: volume
 
-    mop_forbidden_enable:
+Returns: ok or error
 
-    camera_status:
+roborock -d command --device_id aHiddenDeviceId --cmd change_sound_volume --params 72
 
-    is_exploring:
+======================  =========
+Vacuum Model            Supported
+======================  =========
+Roborock S8 Pro Ultra   Yes
+======================  =========
 
-    home_sec_status:
+test_sound_volume
+~~~~~~~~~~~~~~~~~
 
-    home_sec_enable_password:
+Description: Plays a sound on the vacumm to identity volume
 
-    adbumper_status:
+Parameters: None
 
-    water_shortage_status:
+======================  =========
+Vacuum Model            Supported
+======================  =========
+Roborock S8 Pro Ultra   Yes
+======================  =========
 
-    dock_type:
 
-    dust_collection_status:
+get_sound_progress
+~~~~~~~~~~~~~~~~~~
 
-    auto_dust_collection:
+Description:
 
-    avoid_count:
+Parameters:
 
-    mop_mode:
+Returns::
 
-    debug_mode:
+{'sid_in_progress': 0, 'progress': 0, 'state': 0, 'error': 0}
 
-    collision_avoid_status:
+..
+    Is this where the vacumm is currently located ?
 
-    switch_map_mode:
 
-    dock_error_status:
+get_current_sound
```

Comment:
i believ this one and the one above is where you change your voice/ download a new voice